sunpycube/cube/datacube.py

```diff
@@ -75,42 +76,52 @@ def __init__(self, data, wcs, errors=None, **kwargs):
         # Also it's called axes_wcs because wcs belongs to astropy.nddata and
         # that messes up slicing.
 
-    def plot_wavelength_slice(self, offset, axes=None,
-                              style='imshow', **kwargs):
+    def plot_wavelength_slice(self, offset, axes=None, unit_x_axis=None,
+                            unit_y_axis=None, origin='lower', **kwargs):
         """
         Plots an x-y graph at a certain specified wavelength onto the current
         axes. Keyword arguments are passed on to matplotlib.
 
         Parameters
         ----------
-        offset: int or float
+        offset: `int` or `float`
             The offset from the primary wavelength to plot. If it's an int it
             will plot the nth wavelength from the primary; if it's a float then
             it will plot the closest wavelength. If the offset is out of range,
             it will plot the primary wavelength (offset 0)
 
-        axes: matplotlib.axes or None:
+        axes: `astropy.visualization.wcsaxes.core.WCSAxes` or `None`:
             The axes to plot onto. If None the current axes will be used.
 
-        style: 'imshow' or 'pcolormesh'
-            The style of plot to be used. Default is 'imshow'
+        unit_x_axis: `astropy.units.Unit`
+           The unit of x axis.
+
+        unit_y_axis: `astropy.units.Unit`
+            The unit of y axis.
         """
         if axes is None:
-            axes = plt.gca()
+            fig = plt.figure()
```

Comment:
Check the wcsaxes helpers file in sunpy.visualization for a better way to do this.